src/sentry/api/endpoints/source_map_debug.py

```diff
@@ -220,3 +320,24 @@ def _get_releasefiles(self, release, organization_id):
                 data_sources.extend([source[i] for i in range(len(source))])
 
         return data_sources
+
+    def _find_url_prefix(self, filepath, artifact_name):
```

Comment:
@roggenkemper Are we testing the edge cases in existing unit tests? If not, we can add some specific tests just for this method cause I imagine there are some tricky edge cases.

Reply:
will add in some test cases!